src/test/java/space/obminyashka/items_exchange/util/AdvertisementDtoCreatingUtil.java

```diff
@@ -93,10 +79,33 @@ public static String createValidationMessage(String dtoFieldName, String dtoFiel
                 .replace("{max}", maxValidValue);
     }
 
+    public static String createValidationIdMessage(String dtoFieldName, long id, String errorMessage) {
+        return MessageSourceUtil.getMessageSource(errorMessage)
+                .replace("${validatedValue}",
+                        "createAdvertisement.dto." + dtoFieldName + ": " + id);
+    }
+
     public static boolean isResponseContainsExpectedResponse(String expectedResponse, MvcResult mvcResult) throws UnsupportedEncodingException {
         return mvcResult.getResponse().getContentAsString().contains(expectedResponse);
     }
 
+    public static void verifyAdvInternalEntityId(AdvertisementModificationDto dto,
```

Comment:
As I see the method is used only by 2 other tests in the same class. It would be better to move it into it neither keep it here. What do you think?

Reply:
Ok, I can do it, but  as I know it is not a good practice to use private methods in the test classes